REFERENCES.md

```diff
@@ -0,0 +1,329 @@
+
+# Using TypeScript Project References with ts-loader and Webpack
```

Comment:
Lowercase w for webpack please; surprising though it may be, that's the official casing

Reply:
I'll make this change throughout the document.